styles/App.scss

```diff
@@ -1,3 +1,33 @@
-body {
-    font-family: 'Montserrat', sans-serif;
+ @import "./header";
+ @import "./footer";
+ @import "./movie-app";
+
+ html {
+    height: 100%;
+  }
+  html, body {
+    margin: 0;
+    padding: 0;
 }
+  body {
+    font-family: 'Montserrat', sans-serif;
+    background-color: rgb(218, 218, 233);
+    display: grid;
+    min-height: 100%;
+    position: relative;
+
+  }
+  
+  @media screen and (max-width: 500px) {
```

Comment:
We can create mixins in SCSS for these @media screen rules, and also variables for the break points on different devices to have some standard
Like $phone: 728px
etc. etc.